src/components/layout.js

```diff
@@ -28,9 +28,7 @@ const Layout = ({ children }) => (
       <Box backgroundColor="darkBlue" minHeight="100vh">
         <Header siteTitle={data.site.siteMetadata.title} />
 
-        <Box m="0 auto" maxWidth={960} px={1}>
```

Comment:
alright no problem with the max-widths then, but don't change this because this is the margin of other components. shouldn't be a big deal with your change. @psongpin

Reply:
@franreysaycon `but don't change this because this is the margin of other components. shouldn't be a big deal with your change.` sorry but I didn't understand this part. My bad 🤦‍♂️

Reply:
Resolved once Hero PR is merged. @franreysaycon @jromest